server/apps/prepopulate/data_initialization/validators.json

```diff
@@ -41,10 +41,16 @@
         },
         "subject" : {
             "type" : "list",
-            "required" : true
+            "required" : true,
+            "minlength" : 1
```

Comment:
Doesn't the `required` flag already imply a length of at least 1 character? Is this perhaps some validator quirk and should thus be addressed there instead of circumventing it in config?

Just want to make sure, thanks.